<ui version="4.0" >
 <class>devconf</class>
 <widget class="QDialog" name="devconf" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>333</width>
    <height>263</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Device Settings</string>
  </property>
  <layout class="QVBoxLayout" >
   <item>
    <widget class="QCheckBox" name="showIconCheck" >
     <property name="text" >
      <string>Show Tray Icon</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="scriptBox" >
     <property name="title" >
      <string>Execute scripts when device gets...</string>
     </property>
     <layout class="QVBoxLayout" >
      <item>
       <widget class="QCheckBox" name="deactivatedCheck" >
        <property name="text" >
         <string>...deactivated</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="activatedCheck" >
        <property name="text" >
         <string>...activated</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="carrierCheck" >
        <property name="text" >
         <string>...carrier</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="unconfiguredCheck" >
        <property name="text" >
         <string>...carrier and stays unconfigured</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="upCheck" >
        <property name="text" >
         <string>...up</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox" >
     <property name="orientation" >
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons" >
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::NoButton|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>showIconCheck</tabstop>
  <tabstop>deactivatedCheck</tabstop>
  <tabstop>activatedCheck</tabstop>
  <tabstop>carrierCheck</tabstop>
  <tabstop>unconfiguredCheck</tabstop>
  <tabstop>upCheck</tabstop>
  <tabstop>buttonBox</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>devconf</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>248</x>
     <y>282</y>
    </hint>
    <hint type="destinationlabel" >
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>devconf</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>258</x>
     <y>278</y>
    </hint>
    <hint type="destinationlabel" >
     <x>267</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
